Resources/Prototypes/DeltaV/Entities/Mobs/Species/vulpkanin.yml

```diff
@@ -97,6 +97,16 @@
       Female: FemaleVulpkanin
       Unsexed: MaleVulpkanin
   - type: DogVision
+  - type: SizeAttributeWhitelist # Frontier - tall/short traits
+    short: true
+    shortscale: 0.8
+    shortDensity: 140
+    shortPseudoItem: true
```

Comment:
Felinid mains are going to be upset when they find out Smol dogs have their gimmick too LOL. If you're going to do this to Vulps, you might as well do this to Harpies too.